client/pregame/CSavingScreen.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * CSavingScreen.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#include "StdInc.h"
+
+#include "CSavingScreen.h"
+#include "SelectionTab.h"
+
+#include "../CPlayerInterface.h"
+#include "../gui/CGuiHandler.h"
+#include "../widgets/Buttons.h"
+#include "../widgets/TextControls.h"
+
+#include "../../CCallback.h"
+#include "../CGameInfo.h"
+#include "../../lib/CGeneralTextHandler.h"
+#include "../../lib/filesystem/Filesystem.h"
+#include "../../lib/CConfigHandler.h"
+
+CSavingScreen::CSavingScreen()
+	: CSelectionBase(CMenuScreen::saveGame)
+{
+	OBJ_CONSTRUCTION_CAPTURING_ALL;
+	bordered = false;
+	center(pos);
+	// MPTODO: I doubt saving screen need any of it
+	localSi = LOCPLINT->cb->getStartInfo();
+	localMi = new CMapInfo();
+	localMi->mapHeader = std::unique_ptr<CMapHeader>(new CMapHeader(*LOCPLINT->cb->getMapHeader()));
+
+	tabSel = new SelectionTab(screenType); //scenario selection tab
+	tabSel->recActions = DISPOSE; // MPTODO
+	tabSel->recActions = 255;
```

Comment:
This looks strange, first you equal recActions to DISPOSE and then you make it = 255. Is it intentional?

Reply:
Nope, just some remaining from refactoring.